src/Microsoft.AspNetCore.WebHooks.Receivers/Routing/WebHookRouteDataActionDescriptorExtensions.cs

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Mvc.Abstractions;
+using Microsoft.AspNetCore.Routing;
+
+namespace Microsoft.AspNetCore.WebHooks.Routing
+{
+    /// <summary>
+    /// Extension methods with <see cref="ActionDescriptor"/> parameters for the <see cref="RouteData"/> class.
+    /// </summary>
```

Comment:
Doc comments here mainly because I started with `WebHookRouteDataExtensions` and that's `public`. Was somewhat helpful when updating the constraints and may be helpful if we add or change constraints. But LMK if you prefer I remove these.